DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -6,6 +6,10 @@
 
 #include "DataFormats/PatCandidates/interface/liblogintpack.h"
 using namespace logintpack;
+//std::atomic<CovarianceParameterization*> pat::PackedCandidate::covarianceParameterization_(nullptr);
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant